### PR TITLE
mgr/dashboard_v2: Refactor tests code and fix pylint

### PR DESCRIPTION
**Pull request by rjfd**
This PR refactors the unit tests code by creating a helper class that already knows how to load controllers and provides helper methods to make requests.

This PR also fixes the pylint execution and fixes the several errors and warnings

Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

**Comment by rjfd** [comment]
Added commit 5639aa3 from #29 to this PR

File: src/pybind/mgr/dashboard_v2/.pylintrc [hunk 1]
```diff
@@ -126,7 +126,12 @@ disable=print-statement,
         next-method-defined,
         dict-items-not-iterating,
         dict-keys-not-iterating,
-        dict-values-not-iterating
+        dict-values-not-iterating,
+        missing-docstring,
+        invalid-name,
+        no-self-use,
+        too-few-public-methods,
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard_v2/.pylintrc:133]: Hm. which class is affected by `too-few-public-methods`?

**Review by LenzGr — APPROVED** [review]
lgtm - thanks for taking care of that.

**Comment by rjfd** [comment]
Added another commit that merges the coverage information of python 2 and python 3 test results

**Review by sebastian-philipp — APPROVED** [review]
lgtm.

File: src/pybind/mgr/dashboard_v2/.pylintrc (hunk 1 above)
Comment (sebastian-philipp) [reply src/pybind/mgr/dashboard_v2/.pylintrc:133]: <sebastian-w> rdias: If one class has indeed only one method, it IMO shouldn't be a class. 
<sebastian-w> rdias: But in this case,  we have multiple decorators. lgtm

**Pull request merged**